sdks/python/apache_beam/runners/worker/data_plane.py

```diff
@@ -147,6 +148,10 @@ def maybe_flush(self):
   def flush(self):
     # type: () -> None
     if self._flush_callback:
+      if self.size() > _FLUSH_MAX_SIZE:
+        raise ValueError(
+            f'Buffer size {self.size()} exceeds GRPC limit {_FLUSH_MAX_SIZE}. '
+            'This is likely due to a single element that is too large.')
```

Comment:
```suggestion
            'This is likely due to a single element that is too large. '
            'To mitigate, store large elements in external storage systems, '
            'and use PCollections to pass their metadata, '  
            'or use a custom coder that reduces the element's size.')
```

Reply:
IMHO generally elements this large are an anti-patter that are likely to cause other problems (e.g. inability to parallelize).

Reply:
I see. feel free to correct the suggestion.